app/api/helpers/exceptions.py

```diff
@@ -31,3 +33,10 @@ class MethodNotAllowed(JsonApiException):
     """
     title = "Method Not Allowed"
     status = 405
+
+
+def assert_forbidden(argument):
```

Comment:
What use is this for?

Reply:
@iamareebjamal Not sure. I just did things the way you asked. Though I don't think I totally understood the workflow you wanted. So need more insight from you on this.

Reply:
I wanted to remove the duplication